framework/src/controller/bus.js

```diff
@@ -170,6 +170,36 @@ class Bus extends EventEmitter2 {
 		});
 	}
 
+	// TO REMOVE
```

Comment:
Why this code was added and when to be removed? Please refer the issues here.

Reply:
We should start the comment with `TODO: ` so that it is easier to search for it. Also, we should specify when it should be removed; which is as part of this issue: https://github.com/LiskHQ/lisk/issues/2883 (you can put the link in the comment) because this is a hack and we should make sure that nobody on our team will be using this function.